packages/components/spinner/src/Spinner.styles.tsx

```diff
@@ -11,6 +11,7 @@ export const spinnerStyles = cva(
   {
     variants: {
       size: {
+        current: ['w-[1em]', 'h-[1em]'],
```

Comment:
`u-current-font-size` instead ? This way we avoid adding extra hard coded clases 🙏🏻